src/__mocks__/context.ts

```diff
@@ -608,3 +612,16 @@ export interface PartNote {
 	}
 	message: string
 }
+
+export function makeMockAFVDContext(studioConfigOverrides?: Partial<StudioConfig>) {
```

Comment:
It would be nice if we also camelCased abbreviations in our names. 
So that would make this: `makeMockAfvdContext`